recipes/simage/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "simage" %}
+{% set version = "1.7.0" %}
+{% set sha256 = "4b57ed3c4997fbcc8b13823a60c1115cd97f701ada8afd44241f8512a5a9adc0" %}
+{% set download_url = "https://github.com/FreeCAD/pivy/releases/download/0.6.3/simage.tar.gz" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}.tar.gz
+    url: {{ download_url }}
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: true  # [py<35]
+    skip: true  # [osx]
```

Comment:
This is not supported afaik, plese use something like `# [py<35 or osx]` in a single line.